### DISCUSSION TITLE
Clickable elements too close together

### DISCUSSION THREAD

**Discussion by cortix** [mmistakes/minimal-mistakes#3616] [Q&A] [answered]
<img width="1289" alt="Screen Shot 2022-04-29 at 14 52 41" src="https://user-images.githubusercontent.com/6178404/165939222-b10aba44-519e-48b7-9a9c-cca4045508ff.png">

Hi,

When I look at my blog on google search console, for some pages, I am getting 2 warnings for **mobile usability**.

* **Text too small to read**
* **Clickable elements too close together**

I would like to get your help especially for the 2nd warning.

**Comment by iBug** [comment, marked as answer]
Do nothing.

Google occasionally fails to fetch the theme's CSS, which causes these two "issues". It'll self-remediate when Google next crawls your site.

**Reply by cortix** [reply]
I got it. Thank you so much!!!
